docs/source/tutorial/commercial_paper.md

```diff
@@ -167,233 +129,178 @@ window; for example:
 
 indicates that you should run the `ls` command from Isabella's window.
 
-## Create network
+## Create the network
 
-The tutorial currently uses the basic network; it will be updated soon to a
-configuration which better reflects the multi-organization structure of
-PaperNet. For now, this network is sufficient to show you how to develop an
-application and smart contract.
+This tutorial will deploy a smart contract using the Fabric test network.
+The test network consists of two peer organizations and an ordering organization.
+The two peer organizations operate one peer each, while the ordering organization
+operates a single node raft ordering service. We will also use the test network
+to create a single channel named `mychannel` that both peer organizations will
+be members of.
 
-![commercialpaper.network](./commercial_paper.diagram.3.png) *The Hyperledger
-Fabric basic network comprises a peer and its ledger database, an orderer and a
-certificate authority (CA). Each of these components runs as a docker
-container.*
+![commercialpaper.network](./commercial_paper.diagram.testnet.png)
+*The Fabric test network comprises of two peer organizations, Org1 and Org2,
+each with one peer and its ledger database, an ordering node. Each of these
+components runs as a docker container.*
 
-The peer, its [ledger](../ledger/ledger.html#world-state-database-options), the
+The two peers, the peer [ledgers](../ledger/ledger.html#world-state-database-options), the
 orderer and the CA each run in the their own docker container. In production
 environments, organizations typically use existing CAs that are shared with
 other systems; they're not dedicated to the Fabric network.
```

Comment:
I was a little confused later on because there were no CA containers. Perhaps we should mention here 
In production environments, organizations typically use existing CAs that are shared with
other systems; they're not dedicated to the Fabric network. For purposes of this tutorial therefore, 
cryptogen is being used to generate the certs.